transports/config.go

```diff
@@ -5,18 +5,18 @@ import buxmodels "github.com/BuxOrg/bux-models"
 // TransportType the type of transport being used ('http' for usage or 'mock' for testing)
 type TransportType string
 
-// BuxUserAgent the bux user agent sent to the bux server
-const BuxUserAgent = "BUX: go-client " + BuxClientVersion
+// SPVWalletUserAgent the spv wallet user agent sent to the spv wallet.
+const SPVWalletUserAgent = "SPV-Wallet: go-client " + SPVWalletClientVersion
 
-// BuxClientVersion is the version of the client
-const BuxClientVersion = "v0.3.13"
+// SPVWalletClientVersion is the version of the client
+const SPVWalletClientVersion = "v0.3.13"
```

Comment:
hmmm... wonder if this is actually really needed, and if so, then probably you should update it

Reply:
I think we don't need it - I removed version